pkg/gardenlet/controller/shoot/shoot_control_reconcile.go

```diff
@@ -370,6 +370,10 @@ func (c *Controller) updateShootStatusReconcile(o *operation.Operation, operatio
 				shoot.Status.RetryCycleStartTime = retryCycleStartTime
 			}
 
+			if shoot.Status.SeedName == nil {
+				shoot.Status.SeedName = o.Shoot.Info.Spec.SeedName
```

Comment:
We do the assignment here to allow shoots which have errors to be migrated am I right? Or did I miss something?

Reply:
https://github.com/gardener/gardener/pull/2012#discussion_r391434119

Reply:
I've added a doc file under docs/usage/control_plane_migration.md. I think it will be good if you check it, you probably will have some comments.